dell-restore-system-dhc

```diff
@@ -0,0 +1,89 @@
+#! /usr/bin/python3
+# -*- coding: utf-8 -*-
+#
+# Dell Recovery Restore system command line
+#
+# Copyright (C) 2017 Dell Inc,
+#   Author: Mario Limonciello/Jude Hung
```

Comment:
You should probably update this to yourself.

Reply:
Yes, thanks